service/test_automatic_run_experiment.py

```diff
@@ -85,13 +85,24 @@ def test_run_requested_experiment(mocked_get_requested_experiments,
         'zlib_zlib_uncompress_fuzzer',
         'freetype2-2017',
         'harfbuzz-1.3.2',
+        'harfbuzz-1.3.2-antifuzz',
```

Comment:
Sorry, I don't want this change. I need to think about this, but I don't want the anitfuzz benchmarks to be defaults.

Reply:
This part of the change I mean, not talking about the PR as a whole.

Reply:
Okay I will remove it as soon as we know how we should proceed!